src/lib/server/models/discord/attachment/index.ts

```diff
@@ -0,0 +1,17 @@
+import { type InferOutput, nullable, number, object, optional, string } from 'valibot';
+import { Snowflake } from '$lib/server/models/discord/snowflake';
+
+export const Attachment = object({
+    id: optional(Snowflake),
+    filename: string(),
+    title: optional(string()),
+    description: optional(string()),
+    content_type: nullable(optional(string())),
+    size: optional(number()),
```

Comment:
This is apparently required according to the docs.
```suggestion
    size: number(),
```